scripts/deploy-aws.sh

```diff
@@ -1,9 +1,24 @@
 #!/usr/bin/env bash
 
-CONTROLLER=aws-kf-controller
-CLOUD=aws-kf-cloud
-MODEL=aws-kf-model
-AWS_MODEL=default
+for cmd in juju juju-wait kubectl
+do
+    command -v $cmd > /dev/null || {
+    cat << EOF
+The executable \`$cmd\` was not found and is required for this script.
+To install it, run:
+
+    sudo snap install $cmd
+
+EOF
+        exit 1
+    }
+done
```

Comment:
May I suggest the following to report all missing at once, rather than requiring up to four attempts to collect them all:

```bash
missing=$(for cmd in juju juju-wait kubectl; do command -v $cmd > /dev/null || echo $cmd; done)
if [[ -n "$missing" ]]; then
    echo "Please install the required dependencies with:"
    echo
    for cmd in $missing; do
        echo "    sudo snap install $cmd"
    done
    echo
    exit 1
fi
```